### DISCUSSION TITLE
API for login authentication for CLI download

### DISCUSSION THREAD

**Discussion by mfathi91** [gildas-lormeau/SingleFile#927] [General]
Hi,

I need to download and get a backup of all the pull request pages of my code-base in Bitbucket, but when I try to save the page via CLI, it only saves the login page of Bitbucket.

Is there an API to authenticate the user to login? 

Thanks

**Comment by gildas-lormeau** [comment]
You could achieve this by using the `--browser-script` to inject a script that would fill in the login form or by using `--browser-cookie` (or `--browser-cookie-file`) to pass your authentication cookie when processing the page.
